core/core.go

```diff
@@ -793,6 +783,11 @@ func (t *TextileNode) CreateAlbum(mnemonic string, name string) error {
 	return t.Datastore.Albums().Put(album)
 }
 
+// DeleteAlbum removes an album record by id
+func (t *TextileNode) DeleteAlbum(id string) error {
```

Comment:
Wait, why do you want another function for this?

Reply:
ah duh... gone